homeassistant/components/iron_os/entity.py

```diff
@@ -31,11 +32,21 @@ def __init__(
         )
         if TYPE_CHECKING:
             assert coordinator.config_entry.unique_id
-        self.device_info = DeviceInfo(
+
+        serial_number = None
+        if coordinator.config_entry.data.get(CONF_SERIAL_NUMBER):
+            serial_number = f"{coordinator.config_entry.data.get(CONF_SERIAL_NUMBER)} DeviceID:{coordinator.config_entry.data.get(CONF_DEVICE_ID)}"
```

Comment:
what's the device id?

Reply:
it is a unique identifier, that in combination with the serial number is used to verify the authenticity https://pinecil.pine64.org/

Reply:
and maybe I shouldn't have changed the formatting of the serial number string 🤨